src/main/resources/templates/macros/report/element.vm

```diff
@@ -9,12 +9,13 @@
 
   #includeTags($element.getTags())
 
-  <span data-toggle="collapse" class="#if ($element.getElementStatus().isPassed()) collapsed #end collapsable-control" data-target="#element-$element.hashCode()">
+  #set($elementId = $counter.getNext())
+  <span data-toggle="collapse" class="#if ($element.getElementStatus().isPassed()) collapsed #end collapsable-control" data-target="#element-$elementId">
     #includeBrief($element.getKeyword(), $element.getElementStatus(), $element.getName(), true)
   </span>
   <div class="description indention">$element.getDescription()</div>
 
-  <div id="element-$element.hashCode()" class="collapse collapsable-details #if(!$element.getElementStatus().isPassed()) in #end">
+  <div id="element-$elementId" class="collapse collapsable-details #if(!$element.getElementStatus().isPassed()) in #end">
```

Comment:
The way counter is currently used, it's global/unique per page but not report. Since the HTML doesn't reference anything across pages that uses IDs currently, there isn't a conflict.